Properties/Resources.Designer.cs

```diff
@@ -0,0 +1,224 @@
+﻿//------------------------------------------------------------------------------
```

Comment:
A question about _Resources.Designer.cs_ that might be worth a quick discussion is which of the three following options is preferred for this initial LibZipSharp localization PR:

1. Use the output of the generator from Visual Studio for Mac? Advantages: The primary LibZipSharp maintainers work on macOS.
2. Use the output of the generator from Visual Studio on Windows? Advantages: The generators includes `global::` prefixes on `System` types and generates `/// <summary>` blocks on each of the generated properties.
3. Use an alternative generator that doesn't require Visual Studio at all? For example, I recently learned that the Xamarin UITools team uses an alternative that Jérémie adapted (https://github.com/garuma/ResxStrongifier) from another package kzu set up. (Example non-public commit a few other details: <https://github.com/xamarin/UITools/commit/cadbac0177887f4409beabdcb0ccb20dec45fa60>)

So far, xamarin-android, java.interop, and xamarin-android-tools have all gone with option 2. I was imagining that they could transition to option 3 in the future. Originally, I was picturing that xamarin-android could switch to [a generator that's part of dotnet/arcade](https://github.com/dotnet/arcade/blob/72b28b7e023d4c3fffa0a0b9748a7d4e8cc799be/src/Microsoft.DotNet.Arcade.Sdk/src/GenerateResxSource.cs). I haven't yet had a chance to experiment with the dotnet/arcade NuGet package to see if it would be unpleasant to try to use it just for that one generator task, but if it turns out to be easy, that might be an option for LibZipSharp too.

Reply:
to add to this, one of the maintainers works on linux the other on Mac. So I'm not sure what to do about this. 
I do feel however it can wait for later.

Reply:
Perhaps create an issue for this in x-a? We should standardise across all of our repo, since this will be a "thing" once we complete all the localisation.

Reply:
Apologies for the delay on the next review step for this PR. I took a little detour to see if I could indeed set up the dotnet/arcade approach in xamarin-android. It's looking promising: <https://github.com/xamarin/xamarin-android/pull/5270>. If that PR goes through building and testing cleanly, I think the same approach could optionally be applied here too without much trouble (importing a couple files and setting a property).

In either case, I'll plan to get right back to reviewing this LibZipSharp PR tomorrow!